ccc/github.py

```diff
@@ -63,6 +67,31 @@ def github_api_ctor(
         raise ValueError('failed to parse url: ' + str(github_url))
 
     session = github3.session.GitHubSession()
+    original_request = session.request
+
+    def intercepted_request(*args, **kwargs):
+        req = original_request(*args, **kwargs)
+
+        try:
+            method, url = args
```

Comment:
.. how about moving positional parameters to the signature of `intercepted_request`?